ChangeLog

```diff
@@ -119,6 +119,10 @@ API Changes
      mbedtls_ssl_export_keys_ext_t, so that the key exporter is discouraged
      from modifying the client/server hello.
 
+Bugfix
```

Comment:
I don't think this should be here, looks like a duplicate of the entry in version x.x.x.

Reply:
It's not: the original contribution covered both crypto and TLS, the crypto part was merged in time for 2.19.1 but the TLS part wasn't. But that does make the entries confusing. I'll edit them to make them unique.